corehq/form_processor/models.py

```diff
@@ -1203,15 +1203,16 @@ class LedgerValue(DisabledDbMixin, models.Model, TrackRelatedChanges, AbstractLe
     """
     objects = RestrictedManager()
 
-    # domain not included and assumed to be accessed through the foreign key to the case table. legit?
+    domain = models.CharField(max_length=255, null=False, default=None)
     case_id = models.CharField(max_length=255, db_index=True, default=None)  # remove foreign key until we're sharding this
+    location_id = models.CharField(max_length=255, null=True, default=None)
```

Comment:
and this. (not saying we shouldn't add them, but if it's for reporting/efficiency another option could potentially be to dump them into ES but that would be more of a long term plan)